### PR TITLE
CORE-11 Migrate some /apps endpoint schemas.

### PR DESCRIPTION
**Pull request by psarando**
Migrated `/apps/shredder`, `permission-lister`, `sharing`, and `unsharing` docs into the `common-swagger-api` lib.

### CODE REVIEW

File: src/apps/routes/admin.clj [hunk 1]
```diff
@@ -120,7 +121,7 @@
 
   (POST "/shredder" []
     :query [params SecuredQueryParams]
-    :body [body (describe AppDeletionRequest "List of App IDs to delete.")]
```
Comment (psarando) [inline src/apps/routes/admin.clj]: Docs on request bodies like this were made part of the schemas when migrated into `common-swagger-api`.

File: src/apps/routes/apps.clj [hunk 2]
```diff
@@ -26,50 +41,34 @@
...
 
   (POST "/permission-lister" []
     :query [params perms/PermissionListerQueryParams]
-    :body [body (describe perms/QualifiedAppIdList "The app permission listing request.")]
-    :return perms/AppPermissionListing
-    :summary "List App Permissions"
-    :description "This endpoint allows the caller to list the permissions for one or more apps. The
-    authenticated user must have read permission on every app in the request body for this
-    endpoint to succeed."
+    :body [body AppPermissionListingRequest]
```
Comment (psarando) [inline src/apps/routes/apps.clj]: See https://github.com/cyverse-de/common-swagger-api/pull/14#discussion_r269365307

File: project.clj [hunk 3]
```diff
@@ -28,7 +28,7 @@
                  [org.cyverse/metadata-client "3.0.1"]
                  [org.cyverse/common-cli "2.8.1"]
                  [org.cyverse/common-cfg "2.8.1"]
-                 [org.cyverse/common-swagger-api "2.10.2"]
+                 [org.cyverse/common-swagger-api "2.10.3-SNAPSHOT"]
```
Comment (slr71) [inline project.clj]: I just published version `2.10.3`, so this might have to be bumped to `2.10.4-SNAPSHOT`.

**Comment by psarando** [comment]
Thanks for the review! 👍

**Pull request merged**